modules/virt-restarting-vm-web.adoc

```diff
@@ -23,14 +23,12 @@ To avoid errors, do not restart a virtual machine while it has a status of *Impo
 
 * To stay on this page, where you can perform actions on multiple virtual machines:
 
-.. Click the Options menu {kebab} located at the far right end of the row.
+.. Click the Options menu {kebab} located at the far right end of the row and click *Resume VirtualMachine*.
```

Comment:
```suggestion
.. Click the Options menu {kebab} located at the far right end of the row and click *Restart*.
```

Reply:
@ousleyp For this one, I no longer see **Resume VirtualMachine**. However, I see that **Restart** is grayed out. The user needs to click **Start**?? (This is a 4.14.16 cluster.)

![image](https://github.com/openshift/openshift-docs/assets/25209155/9143a7dd-f2be-4a0f-981d-225d777a1705)